Lab3/src/department/Department.java

```diff
@@ -0,0 +1,9 @@
+package department;
+
+import people.Employee;
+
+public interface Department {
```

Comment:
getter and setters should be available on clases, not interafaces, so please leave addEmployee method only here